skydns2/skydns2.go

```diff
@@ -45,7 +45,14 @@ func (r *Skydns2Adapter) Ping() error {
 
 func (r *Skydns2Adapter) Register(service *bridge.Service) error {
 	port := strconv.Itoa(service.Port)
-	record := `{"host":"` + service.IP + `","port":` + port + `}`
+	record := `{"host":"` + service.IP + `","port":` + port
+	if prio, ok := service.Attrs["priority"]; ok {
+		record += `,"priority":` + prio
+	}
+	if weight, ok := service.Attrs["weight"]; ok {
+		record += `,"weight":` + weight
+	}
+	record += `}`
```

Comment:
Since this is building a json string, would it not be better to build the golang struct and then just stringify it?